.gitmodules

```diff
@@ -64,3 +64,9 @@
 [submodule "vendor/golang.org/x/crypto"]
 	path = vendor/golang.org/x/crypto
 	url = https://go.googlesource.com/crypto
+[submodule "vendor/github.com/syndtr/goleveldb"]
+	path = vendor/github.com/syndtr/goleveldb
+	url = https://github.com/syndtr/goleveldb
+[submodule "vendor/github.com/golang/snappy"]
+	path = vendor/github.com/golang/snappy
```

Comment:
what data are we planning to compress?

compression of our data (blocks and such) is a big feature that may not pay off, please discuss with OdedW before introducing compression. or at least I would recommend to avoid compression until we have some realistic data samples that we can check what's the compression ration gain we can get